Corgibytes.Freshli.Cli.Test/Functionality/CacheManagerTest.cs

```diff
@@ -32,7 +32,7 @@ public void SavePersistsACachedAnalysisAndGeneratesAnId()
 
         var cache = cacheManager.GetCacheDb(_tempCacheDir);
 
-        var expectedAnalysis = new CachedAnalysis("https://git.example.com", "main", "1m");
+        var expectedAnalysis = new CachedAnalysis("https://git.example.com", "main", "1m", true);
```

Comment:
I'm not a big fan of boolean values being passed as parameters, because it's always hard to tell what they mean. Can we either change all of these calls to look like:

```suggestion
        var expectedAnalysis = new CachedAnalysis("https://git.example.com", "main", "1m", useCommitHistory: true);
```

Or switch to using an enumeration. Something like `CommitHistory.Full` vs `CommitHistory.Minimal`, where `CommitHistory.Full` is analogous to a value of `true` and `CommitHistory.Minimal` is analogous to a value of `false`.